app/src/main/java/mozilla/lockbox/view/FxALoginFragment.kt

```diff
@@ -40,9 +43,9 @@ class FxALoginFragment : BackableFragment(), FxALoginView {
         view.webView.settings.javaScriptEnabled = true
         CookieManager.getInstance().setAcceptCookie(true)
 
-        if (!isDebug()) {
+        if (!isDebug())
             skipFxA.visibility = View.GONE
-        }
```

Comment:
Nit: keep braces, even for one line blocks.

Reply:
no longer applicable, this piece of code is gone now :)